UnityProject/Assets/Scripts/Core/SecureStuff/MicrophoneAccess.cs

```diff
@@ -0,0 +1,51 @@
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using SecureStuff;
+using UnityEngine;
+
+namespace SecureStuff
+{
+	public static class MicrophoneAccess
+	{
+		private static bool MicEnabled = false;
+
+		public static void End(string deviceName)
+		{
+			Microphone.End(deviceName);
+		}
+
+		public static List<string> GetDevices()
+		{
+			return Microphone.devices.ToList();
+		}
+
+		public static bool IsRecording(string deviceName)
+		{
+			return Microphone.IsRecording(deviceName);
+		}
+
+		public static int GetPosition(string deviceName)
+		{
+			if (MicEnabled == false)
+			{
+				//TODO TEMP
+				//return 0;
```

Comment:
nani

Reply:
wouldn't it be better to do this check wherever this and the other method are called?

Reply:
uhmm, ok. How do you intend to solve this TODO?

Reply:
when hub is pushed out Comment the code